staging/src/k8s.io/apiserver/pkg/server/filters/timeout.go

```diff
@@ -121,6 +122,20 @@ func (t *timeoutHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	case <-after:
 		postTimeoutFn()
 		tw.timeout(err)
+
+		// resultCh needs to have a reader, since the function doing
+		// the work needs to send to it.
+		go func() {
+			res := <-resultCh
+			if res != nil {
+				switch t := res.(type) {
+				case error:
+					utilruntime.HandleError(t)
```

Comment:
I would guess that many timed out requests would end up with eventual errors as the client may have gone away and writing to a closed http connection should return an error.

Reply:
note that this only logs panics, not all errors

Reply:
Yeah I'm really not sure what is in the panics, the situation @cheftako is talking about might result in one, I'm not sure. Anyway I *want* to see these, even if it's spammy; right now I have no idea what they are to either ignore or fix them.

Reply:
agreed